cmd/kola/bootchart.go

```diff
@@ -48,12 +48,14 @@ func runBootchart(cmd *cobra.Command, args []string) {
 		os.Exit(2)
 	}
 
+	var lockfile *os.File
 	var err error
-	outputDir, err = kola.CleanOutputDir(outputDir)
+	outputDir, lockfile, err = kola.CleanOutputDir(outputDir)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Setup failed: %v\n", err)
 		os.Exit(1)
 	}
+	defer lockfile.Close()
```

Comment:
It could be slightly nicer to use CLOEXEC just in case and make sure we dodge any os.Exit calls down the line.

If you want a nice package to do that, https://github.com/rkt/rkt/blob/v1.25.0/pkg/lock/file.go#L160-L190 has code for it already :)